modules/op-about-workspace.adoc

```diff
@@ -0,0 +1,96 @@
+// This module is included in the following assembly:
+//
+// *openshift_pipelines/creating-applications-with-cicd-pipelines.adoc
+
+[id="about-workspaces_{context}"]
+= Workspaces
+
+Workspaces declare shared storage volumes that a Task in a Pipeline needs at runtime. Instead of specifying the actual location of the volumes, Workspaces enable you to declare the filesystem or parts of the filesystem that would be required at runtime. You must provide the specific location details of the volume that is mounted into that Workspace in a TaskRun or a PipelineRun. This separation of volume declaration from runtime storage volumes makes the Tasks reusable, flexible, and independent of the user environment.
+
+Listed below are the various ways in which you can use Workspaces:
+
+* Store Task inputs and outputs
+* Share data among Tasks
+* Use it as a mount point for credentials held in Secrets
+* Use it as a mount point for configurations held in ConfigMaps
+* Use it as a mount point for common tools shared by an organization
+* Create a cache of build artifacts that speed up jobs
+
+You can specify Workspaces in the TaskRun or PipelineRun using:
+
+* A read-only ConfigMaps or Secret
+* An existing PersistentVolumeClaim shared with other Tasks
+* A PersistentVolumeClaim from a provided VolumeClaimTemplate
+* An emptyDir that is discarded when the TaskRun completes
+
+Here is a code snippet of the `build-and-deploy` Pipeline which declares a `shared-workspace` Workspace for the Tasks, `build-image` and `apply-manifests`, defined in the Pipeline.
```

Comment:
`build-image` , and

Reply:
I think we can avoid this comma, as there is a comma just before `build-image`.